examples/es6_output/es6_output_test.sh

```diff
@@ -2,15 +2,15 @@
 set -e
 
 # should not down-level ES2015 syntax, eg. `class`
-readonly GREETER_JS=$(cat $TEST_SRCDIR/build_bazel_rules_typescript/examples/es6_output/greeter.closure.js)
+readonly GREETER_JS=$(cat $TEST_SRCDIR/build_bazel_rules_typescript/examples/es6_output/es6_output.prod/examples/es6_output/greeter/greeter.js)
```

Comment:
shouldn't this be `es6_output_lib.prod` ?

Reply:
No, we use the label of the bundling rule rather than each individual ts_library since we are rerooting under a single tree rather than many.